src/components/formControls/input/index.tsx

```diff
@@ -3,28 +3,48 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
-import { StyledInput } from './style'
+import { InputComponent, StyledIcon, StyledInput } from './style'
 
 export interface Props {
-  id?: string
-  defaultValue?: string
+  testId?: string
+  icon?: React.ReactNode
+  type?: 'text' | 'email' | 'search' | 'password'
```

Comment:
I think that we need to add text as default.

More info here https://github.com/brave/brave-ui/wiki/Code-style-guide#default-props